Fabric.Authorization.AccessControl/src/app/modules/access-control/access-control.module.ts

```diff
@@ -5,15 +5,28 @@ import { AccessControlRoutingModule } from './access-control-routing.module';
 import { UserlistComponent } from './userlist/userlist.component';
 import { UseraddComponent } from './useradd/useradd.component';
 import { AuthserviceService } from '../../services/authservice.service';
+import { HttpClientModule } from '@angular/common/http';
```

Comment:
we may have to adjust our setup and not import this here. we need to sort out the interceptor stuff though i believe